t/plugin/traffic-split.t

```diff
@@ -1150,3 +1144,75 @@ GET /t
 1980, 1981, 1981, 1982, 1982
 --- no_error_log
 [error]
+
+
+
+=== TEST 34: multiple upstream
```

Comment:
Better to add a test for "multiple upstream + empty_upstream is used"

Reply:
updated.